src/lib/files.js

```diff
@@ -147,6 +147,29 @@ export async function getShareableLink (files, gatewayUrl, ipfs) {
   return `${gatewayUrl}/ipfs/${cid}${filename || ''}`
 }
 
+/**
+ *
+ * @param {FileStat[]} files
+ * @param {string} apiUrl
+ * @param {IPFSService} ipfs
+ * @returns {Promise<FileDownload>}
+ */
+export async function getDagCarLink (files, apiUrl, ipfs) {
+  let cid
+
+  if (files.length === 1) {
+    cid = files[0].cid
+  } else {
+    cid = await makeCIDFromFiles(files, ipfs)
+  }
+
+  return {
+    url: `${apiUrl}/api/v0/dag/export?arg=${cid}`,
```

Comment:
@hacdias go-ipfs 0.13 will allow for fetching CAR from gateway, mind refactoring this, so we use below?

```suggestion
    url: `${gatewayUrl}/ipfs/${cid}?format=car&filename=${filename || cid}.car`,
```

(would be cool if we could preserve filename, if not, cid is fine too)